assets/cluster-bootstrap/ingress-to-route-controller-clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:openshift-controller-manager:ingress-to-route-controller
+roleRef:
```

Comment:
@csrwng I noticed that `roleRef` is missing `apiGroup: rbac.authorization.k8s.io` and causes apply via `kubectl` to fail.  However an apply with `oc` works.  Is this something that we should fix?